saltstack/base/salt/prerequisites/user_uid.sls

```diff
@@ -0,0 +1,34 @@
+{% set ids = salt['grains.filter_by']({
+    'amazon-ebs': {
+        'cloudera_scm_user': '992',
+        'cloudera_scm_group': '988',
+    },
+    'azure-arm': {
+        'cloudera_scm_user': '991',
+        'cloudera_scm_group': '987',
+    },
+    'googlecompute': {
+        'cloudera_scm_user': '992',
+        'cloudera_scm_group': '988',
+    },
+},
+grain='builder_type',
+default='amazon-ebs'
+)%}
+
+create_cloudera_scm_group:
+  group.present:
+    - name: cloudera-scm
+    - gid: {{ ids.cloudera_scm_group }} 
```

Comment:
What happens on the YCloud image?

Reply:
It gets the default value, which is in this case, AWS. Do we have prewarmed YCloud images to check for the uid?

Reply:
No, I just wanted to make sure the base image doesn't fail.